main.go

```diff
@@ -79,19 +79,27 @@ func main() {
 		submarinerInformers.WithNamespace(submSpec.Namespace))
 
 	start := func(context.Context) {
+		var subnets []string
+
 		localCluster, err := util.GetLocalCluster(submSpec)
 		if err != nil {
 			klog.Fatalf("Fatal error occurred while retrieving local cluster from %#v: %v", submSpec, err)
 		}
 
+		if len(submSpec.GlobalCidr) > 0 {
```

Comment:
Are you planning to propose a patch in helm/operator that supplies the environment variable (SUBMARINER_GLOBALCIDR) during initialization?

f.e., https://github.com/submariner-io/submariner-charts/blob/e1a313755ceba38c35b0da40f3aa63acb28562d4/submariner/templates/engine-deploy.yaml#L59

Reply:
@mkolesnik is working on that.